weblate/formats/exporters.py

```diff
@@ -367,3 +368,13 @@ class XlsxExporter(CVSBaseExporter):
     def serialize(self):
         """Return storage content."""
         return XlsxFormat.serialize(self.storage)
+
+
+@register_exporter
+class JsonExporter(BaseExporter):
+    name = "json"
+    content_type = "application/json"
+    extension = "json"
+    verbose = _("JSON")
+    storage_class = JsonFile
```

Comment:
This `JsonFile` class is doing all the heavy lifting here. It's already set up to format a translation into a json file.